src/PrestaShopBundle/Resources/views/Admin/Product/form.html.twig

```diff
@@ -783,7 +783,7 @@
                         <div class="alert alert-info alert-down" role="alert">
                           <p class="alert-down-text">
                             {{ 'Choose terms and keywords that your customers will use to search for this product and make sure you are consistent with the tags you may have already used.'|trans({}, 'Admin.Catalog.Help')|raw }}<br>
-                            {{ 'You can manage tag aliases in the [1]Search section[/1].'|trans({}, 'Admin.Catalog.Help')|
+                            {{ 'If you add new tags, you have to rebuild the index. You can manage tag aliases in the [1]Search section[/1].'|trans({}, 'Admin.Catalog.Help')|
```

Comment:
You can manage tag aliases in the [1]Search section[/1]. If you add new tags, you have to rebuild the index.